### PR TITLE
Add translations and fixes

### PR DESCRIPTION
**Pull request by Imzxhir**
Description: This PR adds translations to the wiki with instructions on how to get setup up. This PR also includes many fixes to the docs and replaces a few things.

I have also added stuff to the README file for people who want to build the docs locally and maybe contribute anything if they want to. I think this helps with bringing in people who want to help with the docs, but are not sure how.

If you want to have a look at the docs with these changes, here is a link to it: <https://sidestore-docs.netlify.app/>

### CODE REVIEW

**Review by naturecodevoid — APPROVED** [review]
LGTM

File: src/content/i18n/fr.json
Comment (Dadoum) [inline src/content/i18n/fr.json:1]: I would probably have used some different expressions here, like that
```json
{
  "skipLink.label": "Sauter au contenu",
  "search.label": "Rechercher",
  "search.shortcutLabel": "(Appuyez sur / pour rechercher)",
  "search.cancelLabel": "Annuler",
  "search.devWarning": "La recherche n'est disponible qu'en mode production. \nVeuillez construire et prévisualiser votre site pour tester la recherche localement.",
  "themeSelect.accessibleLabel": "Sélectionner le thème",
  "themeSelect.dark": "Sombre",
  "themeSelect.light": "Clair",
  "themeSelect.auto": "Auto",
  "languageSelect.accessibleLabel": "Sélectionner la langue",
  "menuButton.accessibleLabel": "Menu",
  "sidebarNav.accessibleLabel": "Naviguer sur le wiki",
  "tableOfContents.onThisPage": "Sur cette page",
  "tableOfContents.overview": "Début",
  "i18n.untranslatedContent": "Ce contenu n’a pas encore été traduit dans votre langue.",
  "page.editLink": "Modifier cette page",
  "page.lastUpdated": "Dernière mise à jour :",
  "page.previousLink": "Précédent",
  "page.nextLink": "Suivant",
  "404.text": "Page introuvable. Veuillez revérifier l’URL ou utilisez la barre de recherche."
}
```

Comment (Imzxhir) [reply src/content/i18n/fr.json:1]: These values only provide initial translations for the Starlight UI, so the the user does not have to translate them. I have got them from the Starlight repo which is linked [here](https://github.com/withastro/starlight) if you want to check them out.

The place where I got them from is [here](https://github.com/withastro/starlight/tree/main/packages/starlight/translations).

**Comment by aaronliu0130** [comment]
Excuse my ignorance, but how do I view the sidestore-docs.netlify.app in a different language? I can't find a selection anywhere 😅

**Comment by Imzxhir** [comment]
> Excuse my ignorance, but how do I view the sidestore-docs.netlify.app in a different language? I can't find a selection anywhere 😅

All good. The actual translations have not been added yet, since they are to be done by the community and anyone who wants to contribute. I have made instructions on how to translate the wiki if you go to <https://sidestore-docs.netlify.app/> and click 'Translating the wiki' on the sidebar.

File: src/content/i18n/fr.json
Comment (nythepegasus) [reply src/content/i18n/fr.json:1]: @Dadoum I presume you're a "native" French speaker and would probably have a better idea of how to translate those phrases, but I think for now just to get the ball rolling I will merge and we can revisit getting more accurate translations from those who for sure know the languages within the community.

**Comment by aaronliu0130** [comment]
Hmm, what would the language selector look like if there were translations?

**Comment by Imzxhir** [comment]
It would look like this:

image

**Pull request merged**